packages/web/src/pages/sign-up-page/pages/SelectArtistsPage.tsx

```diff
@@ -94,10 +94,17 @@ export const SelectArtistsPage = () => {
     })
 
   const artists = isFeaturedArtists ? featuredArtists : topArtists
+
   const isLoading =
     (isFeaturedArtists ? featuredArtistsStatus : topArtistsStatus) ===
     Status.LOADING
 
+  // Note: this doesn't catch when running `web:prod`
+  const devEnvironment = process.env.NODE_ENV
```

Comment:
I think instead we should pull env from 

'services/env' and do ENVIRONMENT === 'development' e.g. https://github.com/AudiusProject/audius-protocol/blob/c271e9b1ad4f4133e2fad1261779f2dc34bcde27/packages/common/src/services/audius-backend/AudiusBackend.ts#L754

instead of looking at NODE_ENV

Reply:
this means if you did build:dev and pushed it somewhere it would still work

Reply:
Nice, I dig it. Just changed a few things to use this and it's looking good! 👍